### ISSUE TITLE
if there are no errors, ansible-lint-junit is reporting 1 test and 1 error 

### ISSUE THREAD

**Issue by themr0c** [wasilak/ansible-lint-junit#4]
Documentation is stating that "if there are no errors, empty JUnit XML will be created".

But if there are no errors, ansible-lint will produce an empty file. 

And ansible-lint-junit is reporting 1 test and 1 error when an empty file is provided as input:

```rm empty.txt 
touch empty.txt
ansible-lint-junit  --version
ansible-lint-junit 0.9
ansible-lint-junit  empty.txt  -v 
<?xml version='1.0' encoding='utf8'?>
<testsuites><testsuite errors="1" failures="0" tests="1" time="0" /></testsuites>
```

**Comment by themr0c** [comment]
Same for ansible-lint-junit 0.10.
